app/controllers/api/base_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Api
+  class BaseController < ApplicationController
+    before_action :set_paper_trail_whodunnit
+    before_action :authenticate_user!
+    before_action :set_raven_context
+
+    skip_before_action :verify_authenticity_token
+
+    private
+
+    def authenticate_user!
+      render json: {}, status: :unauthorized unless current_user
```

Comment:
do we need to overwrite this ? By default authenitcate_user! should return unauthorized code